frameworks/kafka/tests/client.py

```diff
@@ -2,14 +2,14 @@
 A collection of client utilites for Kafka.
 """
 import logging
+from toolz import merge as merge_dictionaries
```

Comment:
Nit: We could call `toolz.merge` from `sdk_utils.merge_dictionaries` instead.

Alternatively: Since we're making the changes on every line anyway, we could just call `toolz.merge` directly too.

Reply:
I thought we wanted to keep `sdk_utils` for DC/OS specifics? https://github.com/mesosphere/dcos-commons/pull/2511#discussion_r207285221